src/HHbbVV/postprocessing/postprocessing.py

```diff
@@ -392,6 +411,31 @@ def main(args):
         )
         print("Loaded BDT preds\n")
 
+    # Load VBF Variables (edits events_dict so that all of the events have the appropriate variables and stuff) TODO:
+    if args.vbf:
+        pt_labels = ["JES", "JER"]
+        m_labels = ["JMS", "JMR"]
+        for key, df in events_dict.items():
+            # for each JES,JER, and JMS JMR correction
+            ptlabel = "_JES_up"
+            mlabel = "_JMS_down"
+
+            bb_mask = (
+                df[("ak8FatJetParticleNetMD_Txbb", 0)] > df[("ak8FatJetParticleNetMD_Txbb", 1)]
+            )
+
+            if key == "Data":
```

Comment:
won't this mean you're not saving the nominal (non up/down) values for non-Data?

Reply:
Ya, oops I thought I had pushed an edit for this but the if key == "Data": part gets turned into  "_add_vbf_columns(df, bb_mask, ptlabel="", mlabel="")

            if key == "Data":
                continue"

Reply:
I can add that it